library/packages/src/lib/y2packager/product_upgrade.rb

```diff
@@ -41,7 +41,10 @@ class ProductUpgrade
       # this one is used when openSUSE is not available, e.g. booting SLE medium
       # (moreover the openSUSE medium should contain only one product so that
       # product should be used unconditionally)
-      ["openSUSE"]                                                        => "SLES"
+      ["openSUSE"]                                                        => "SLES",
+      # (installed) openSUSE Leap => (available) SLES,
+      # same as above, in 15.3+ the product has been renamed from "openSUSE" to "Leap"
+      ["Leap"]                                                            => "SLES"
```

Comment:
don't we need also openSUSE => Leap?

Reply:
Good question... But as we got no bug reports I hope we do not need that. The upgrade is covered quite well in openQA...

Reply:
And this is mainly for the SCC migration, you cannot do Leap -> Leap upgrade via SCC...